libvast/builtins/formats/zeek_tsv.cpp

```diff
@@ -670,6 +666,12 @@ auto parser_impl(generator<std::optional<std::string_view>> lines,
         co_return;
       }
     }
+    const auto now = std::chrono::steady_clock::now();
+    if (b->rows() >= defaults::import::table_slice_size
+        or last_finish + std::chrono::seconds{1} < now) {
```

Comment:
This should probably be a named constant under `defaults::`

Reply:
We have a story for implementing this more generically for all parsers in the future.